### DISCUSSION TITLE
Problem to connect to Cloudflare R2 bucket.

### DISCUSSION THREAD

**Discussion by meera** [iterate-ch/cyberduck#16515] [Q&A]
- I am unable to make it work to connect to Cloudflare R2 bucket .

I tried creating a new profile ( The profile creation unfortunately has no 'OK' button so it is a very confusing UI. 

I managed to provide the URL/ Access Key ID/  Secrete Key

This is what it shows.  Please advise.

![image](https://github.com/user-attachments/assets/61b90aa1-ad4d-4f7a-aa4c-1ae17b24fb10)

**Comment by dkocher** [comment]
Can you try to use the _CloudFlare R2_ connection profile available from _Preferences → Profiles_. After enabling the profile make sure to select it as the protocol to use by editing your existing bookmark.

**Reply by dkocher** [reply]
Additionally as indicated in our [documentation](https://docs.cyberduck.io/protocols/s3/cloudflare/), ensure you configure a bucket in _Path_ if your credentials are limited to Object Read & Write or Object Read only.
